bower.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
until https://github.com/heroku/heroku-buildpack-emberjs/issues/24 is addressed we will still need an empty version of this file to exist in the project for the buildpack to deploy the app successfully.  You can leave the existing file in place and just remove the `dependencies` section.